CraftTweaker2-API/src/main/java/crafttweaker/api/util/IRandom.java

```diff
@@ -0,0 +1,24 @@
+package crafttweaker.api.util;
+
+import crafttweaker.annotations.ZenRegister;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.util.IRandom")
+@ZenRegister
+public interface IRandom {
```

Comment:
can you also add the methods that are found in MathHelper that take a rand as a parameter - for example, the nextX with min and max bound methods

Reply:
(these are the `getInt`, `getRandomUUID`, `nextFloat`, and `nextDouble` methods, specifically)